drivers/metal/rendering_context_driver_metal.mm

```diff
@@ -159,7 +159,7 @@ Error resize(uint32_t p_desired_framebuffer_count) override final {
 		return RDD::FramebufferID(&frame_buffer);
 	}
 
-	void present(MDCommandBuffer *p_cmd_buffer) override final {
+	void present(MDCommandBuffer *p_cmd_buffer, double present_minimum_duration) override final {
```

Comment:
By overriding the `RenderingDeviceDriver::swap_chain_set_max_fps` virtual method , you won't need to pass in the value, as it should be available on this object.